lib/arrow.cpp

```diff
@@ -1156,7 +1200,7 @@ namespace grnarrow {
         std::string field_name(column_name, column_name_size);
         auto range_id = grn_obj_get_range(ctx_, column);
         auto range = object_cache_[range_id];
-        auto field_type = grn_column_to_arrow_type(ctx_, range, object_cache_);
+        auto field_type = grn_column_to_arrow_type(ctx_, range, object_cache_, ArrowArrayOutputMode::PLAIN);
```

Comment:
We can add API to get the current total key size. For example, we can use `grn_hash_total_key_size()` for `grn_hash`.

If total key size isn't large, should we use dictionary?

Reply:
Sounds good.
I would like to add it, but is it okay to respond with another pull request?

Reply:
Yes. We should work on it (adding `grn_table_total_key_size()`) by another pull request.

Reply:
OK, I have created a new issue.
https://github.com/groonga/groonga/issues/1266